src/components/inputs/Select/single.js

```diff
@@ -33,9 +33,18 @@ const SingleSelect = ({
     returnFullObject,
     helperText,
 }) => {
-    const intl = useSafeIntl();
+    const { formatMessage } = useSafeIntl();
     const classes = useStyles();
 
+    const displayedErrors = useMemo(() => {
+        const tempErrors = [...errors];
+        const missingValueError = !getOption(value, options);
+        if (value && !loading && missingValueError) {
+            tempErrors.push(formatMessage(MESSAGES.valueNotFound));
```

Comment:
In the multi, you display the value that is not found, I think they should both do the same